Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:16.04
```

Comment:
Can we have a `docker` directory and put the Dockerfile in that?

Reply:
Hmm good idea lat me take a look on that !